protocol-specs/generator_outputs/lisk_codec/validBytesEncodings.json

```diff
@@ -15,31 +15,7 @@
 			"input": {
 				"bytes": {
 					"object": {
-						"address": {
-							"type": "Buffer",
-							"data": [
-								225,
-								26,
-								17,
-								54,
-								71,
-								56,
-								34,
-								88,
-								19,
-								248,
-								110,
-								168,
-								82,
-								20,
-								64,
-								14,
-								93,
-								176,
-								141,
-								110
-							]
-						}
+						"address": "á\u001a\u00116G8\"X\u0013øn¨R\u0014@\u000e]°n"
```

Comment:
maybe we could have an example without control characters in it? or the string of the hex of the bytes? I find this to be a bit weird. Also, how is that 5 bytes?
If this is how the protocol specs are written, then that's fine.

Reply:
No strong opinion on it. @shuse2 suggested binary but we could also use hex for this. Comments?

Reply:
Either way it's fine for me

Reply:
but is that only 5 bytes?

Reply:
Changed to hex to help with readability. Resolving.